Python 3.7 test failure TestTimeZone  ValueError not raised by timezone

Debian got now Python 3.7 and updated package fails the test with
```
    def test_constructor(self):
        self.assertIs(timezone.utc, timezone(timedelta(0)))
        self.assertIsNot(timezone.utc, timezone(timedelta(0), 'UTC'))
        self.assertEqual(timezone.utc, timezone(timedelta(0), 'UTC'))
        # invalid offsets
        for invalid in [timedelta(microseconds=1), timedelta(1, 1),
                        timedelta(seconds=1), timedelta(1), -timedelta(1)]:
>           self.assertRaises(ValueError, timezone, invalid)
E           AssertionError: ValueError not raised by timezone
```
only in 3.7, ok in 3.6.  This is a 1.1.0 release version (with few patches to ignore other failing tests, e..g #663  etc)

--- a/.travis.yml
+++ b/.travis.yml
@@ -39,6 +39,13 @@ matrix:
     - env: DISTRIB="conda" PYTHON_VERSION="3.6" NUMPY=true  PILLOW=true  JPEG2000=false JPEG_LS=true  GDCM=false
     - env: DISTRIB="conda" PYTHON_VERSION="3.6" NUMPY=true  PILLOW=true  JPEG2000=false JPEG_LS=true  GDCM=true
 
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=false PILLOW=false JPEG2000=false JPEG_LS=false GDCM=false
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=true  PILLOW=false JPEG2000=false JPEG_LS=false GDCM=false
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=true  PILLOW=false JPEG2000=false JPEG_LS=false GDCM=true
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=true  PILLOW=true  JPEG2000=false JPEG_LS=false GDCM=false
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=true  PILLOW=true  JPEG2000=false JPEG_LS=true  GDCM=false
+    - env: DISTRIB="conda" PYTHON_VERSION="3.7" NUMPY=true  PILLOW=true  JPEG2000=false JPEG_LS=true  GDCM=true
+
     - env: DISTRIB="pypy" PYTHON_VERSION="2.7" NUMPY=false
     - env: DISTRIB="pypy" PYTHON_VERSION="2.7" NUMPY=true
     - env: DISTRIB="pypy" PYTHON_VERSION="3.5" NUMPY=false

--- a/pydicom/tests/test_fixes.py
+++ b/pydicom/tests/test_fixes.py
@@ -12,7 +12,10 @@ from datetime import datetime
 from datetime import timedelta
 from datetime import tzinfo
 
+import pytest
+
 import pydicom as pydicom_module
+from pydicom import compat
 from pydicom.util.fixes import timezone
 
 pickle_choices = [(pickle, pickle, proto)
@@ -82,6 +85,8 @@ class USTimeZone(tzinfo):
 Eastern = USTimeZone(-5, "Eastern",  "EST", "EDT")
 
 
+@pytest.mark.skipif(not compat.in_py2,
+                    reason='only test the backport to Python 2')
 class TestTimeZone(unittest.TestCase):
     """Backport of datetime.timezone tests.
 

